bin/todo.pl

```diff
@@ -255,9 +276,44 @@ sub mark_done {
     die "Unable to mark as done\n";
 }
 
-## no critic 'sigal'
-sub help { return &unsupported }
-## use critic
+sub help {
+    my ($config, @action_names) = @_;
+
+    if ( @action_names ) {
+	for my $an ( @action_names ) {
+	    my $addon = Text::Todo::Addon::for_name( $config, $an );
+	    if ( $addon ) {
+		$addon->( 'usage' );
+	    } elsif ( exists $actions{ $an } ) {
+		my $ah = Text::Todo::Help::action_help( $an );
+		print "$$ah\n";
+	    } else {
+		die "TODO: No action \"${an}\" exists.\n";
+	    }
+	}
+	return;
+    }
+
+    # use a pager if one is available
```

Comment:
This pager code was what gave me pause on merging this.  I am not a super fan, but based on the code in perldoc there isn't really a better solution that I know of.